src/query/interpreter.cpp

```diff
@@ -328,7 +328,7 @@ class ReplQueryHandler {
           .port = static_cast<uint16_t>(*port),
       };
 
-      if (!handler_->SetReplicationRoleReplica(config, std::nullopt)) {
+      if (!handler_->TrySetReplicationRoleReplica(config, std::nullopt)) {
```

Comment:
can we make std::nullopt default arg

Reply:
No, since it is overload of virtual methods and virtual methods can't have default arguments